src/components/editor/PageUpdater.js

```diff
@@ -79,7 +83,7 @@ export default class PageUpdater {
   async _updateDatabase() {
     function removeFromStringify(data) {
       return JSON.stringify(data, (key, value) => {
-        if (key === 'document_version') return undefined;
+        if (key === 'splash_md') return undefined;
```

Comment:
Why are we removing splash_md? What is splash_md?

Reply:
splash_md is the metadata object on a splash document. We are removing it because when we are updating a document we don't include any of the metadata.